script/Common/00_00.txt

```diff
@@ -954,75 +958,75 @@ The woman said that with her arms crossed, brimming with confidence.
 [sha:2b6310608fe3c6521063b2010731b72092b1023c]{
 -- Page 64, Offset 290.
 -- ともだちみたいな気軽さで、女の人は手を差し伸べてきた。
-The woman cheerily held out her hand to me, as though we were already friends.
+The woman cheerily holds out her hand to me, as though we are already friends.
 }
 [sha:c6f02d5ae5c4a826c9c72982fef983fca0a02bc0]{
 -- Page 64, Offset 291.
 -- 断る理由も見当たらなくて、僕は遠野志貴と名前をつげて、吸いこまれるように手のひらを握り返した。
-I couldn't find any reason to refuse, so I told her my name was Tohno Shiki and let her outstretched hand engulf mine.
+I can't find any reason to refuse, so I tell her my name is Tohno Shiki and let her outstretched hand engulf mine.
 }
 [sha:a926f5bc85f15c1f51eaa894814f3f083449882d]{
 -- Page 65, Offset 292.
 -- 女の人とのおしゃべりはとても楽しかった。
-I enjoyed chatting with her quite a lot.
+I enjoy chatting with her quite a lot.
 }
 [sha:224c273d69aad11aa518c161567032a19e7c8458]{
 -- Page 65, Offset 293.
 -- この人は僕の言うことを『子供だから』と無視しない。
-She didn't dismiss what I said just because I was a child.
+She doesn't dismiss what I say just because I am a child.
 }
 [sha:750538c070fa0928124e43bcdc3850b3f884f90e]{
 -- Page 65, Offset 294.
 -- ちゃんとひとりの人間として、僕の話を聞いてくれた。
-She listened to me thoughtfully, as another human being.
+She listens to me thoughtfully, as any human would.
```

Comment:
Is this correct? Is ちゃんとひとりの人間として not Shiki saying that Aoko treated him as a real human being, listening to him and understanding him instead of just asking about how he broke the bed?

Reply:
Aoko herself is the ちゃんとひとりの人間として, since she's the once doing the action 聞いてくれた.